### DISCUSSION TITLE
Version 1.12.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1921] [Changelog]
## Bugfixes
- #1889 better-featured-project blurs profile page banner. (prio-4)
- #1912 scratch-messaging lacks zalgo protection. (prio-4)
- #1919 editor-devtools context menu has inconsistent capitalization. (prio-4)
- #1919 editor-devtools adds "send to top" button on unintended places. (prio-4)
- #1925 mute-project cannot be triggered on macOS. (prio-3)
- #1947 folders causes crashes when backpacking a folder. (prio-2)
- #1955 scratchstats uses old API for follower history. (prio-4)
- #1980 hide-flyout animation is broken. (prio-3)

## New Features
- #1768 New addon: 2d-color-picker (prio-3)
- #1773 New addon: custom-block-shape (prio-3)
- #1781 Add transparency support to settings page color picker. (prio-3)
- #1830 scratchstats: show message when ScratchDB is down (prio-4)
- #1831 full-signature: add scratchblocks scroll support (prio-4)
- #1889 better-featured-project: add blur option (prio-4)
- #1916 New addon: drag-drop (prio-3)
- #1931 New addon: dango-rain (prio-3)
- #1951 variable-manager: allow renaming variables (prio-3)

**Comment by Exlectruz** [comment]
YESSSSSSSSSS TRANSPARENCY PICKERRRR!!!

**Comment by Endermaster3690** [comment]
when will it release?

**Reply by Endermaster3690** [reply]
never mind its in milestones maybe in march 26th
